packages/contracts/package.json

```diff
@@ -37,7 +37,7 @@
     "pregenTS": "shx mkdir -p dist",
     "precommit": "#lint-staged",
     "compile": "truffle compile",
-    "build:contract": "yarn compile && yarn genTS && tsc --build tsconfig.build.json && yarn prettier --write src/**/*",
+    "build:contract": "yarn compile && yarn genTS && tsc --build tsconfig.build.json && yarn prettier --write src/**/* && shx cp -r src/contracts dist",
```

Comment:
> 1. add zkopru-network/zkopru as a submodule into stress-test repo

Yeah, That is my next step after this PR merged! Thanks.

> Move genesis.json & run_geth.sh file (or other compose files if needed) to `stress-test` repo

I forgot to remove these files in `zkopru/packages/contract`. Chance already moved the files to `stress-test` repo.

Reply:
thx @sifnoc how do you think about the comment about the `dist` folder commands?

Reply:
> dist directory is for the distribution and we're not planning to include solidity files inside the dist folder.

`stress-test` could not build without this, for now. I am trying to find another way to build without this copy.

Reply:
> thx @sifnoc how do you think about the comment about the dist folder commands?

I just realized that the `dist` folders are using for running containers. This is a simple way to build `stress-test` but it makes unnecessary files are included in a container image.

I am still working on this, to build `stress-test` without this. :)

Reply:
Found and updated at 938f0e0

Reply:
I tried to convert `contracts/src/contracts/*.d.ts` to `.ts` with newly added script in utlis. But it makes more time to build in `zkopru` repo. So It would be better to treat this in `stress-test` repo without using the copy command or something similar in `zkopru` repo.